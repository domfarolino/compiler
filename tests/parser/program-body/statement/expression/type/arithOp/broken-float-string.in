program dominic is

  float result;

begin

  result := 10.592 + "dominic";

end program.
