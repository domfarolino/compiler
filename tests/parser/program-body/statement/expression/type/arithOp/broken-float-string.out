[1;31mLine 7: [ArithOp]: float and string cannot be added or subtracted
Line 7: Error parsing program body
[0m